AdobeIms/Model/Config.php

```diff
@@ -25,6 +25,7 @@ class Config implements ConfigInterface
     private const XML_PATH_LOGOUT_URL_PATTERN = 'adobe_ims/integration/logout_url';
     private const XML_PATH_DEFAULT_PROFILE_IMAGE = 'adobe_ims/integration/default_profile_image';
     private const XML_PATH_IMAGE_URL_PATTERN = 'adobe_ims/integration/image_url';
+    private const OAUTH_CALLBACK_URL = 'adobe_ims/oauth/callback';
```

Comment:
Please prefix the constant with `XML_PATH_`

Reply:
I don't think this prefix is needed. That's an actual URL path, not the configuration path